### PR TITLE
[PLT-6496] Remove System Console and config settings for image height and width

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Remove System Console and config settings for image height and width

#### Ticket Link
Jira ticket: [PLT-6496](https://mattermost.atlassian.net/browse/PLT-6496)

#### Checklist
- [x] All new/modified APIs include changes to the drivers
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

File: app/file.go [hunk 1]
```diff
@@ -553,18 +553,16 @@ func getImageOrientation(input io.Reader) (int, error) {
 }
 
 func generateThumbnailImage(img image.Image, thumbnailPath string, width int, height int) {
-	thumbWidth := float64(utils.Cfg.FileSettings.ThumbnailWidth)
-	thumbHeight := float64(utils.Cfg.FileSettings.ThumbnailHeight)
-	imgWidth := float64(width)
-	imgHeight := float64(height)
+	thumbWidth := 120
+	thumbHeight := 100
```
Comment (jwilander) [inline app/file.go]: Can you change these to be constants at the top of the file? Something like `IMAGE_THUMBNAIL_PIXEL_WIDTH` Makes it a little clearer.

Also please make the same changes for the other instances?

File: webapp/components/admin_console/admin_sidebar.jsx [hunk 2]
```diff
@@ -528,24 +528,18 @@ export default class AdminSidebar extends React.Component {
...
+                                        key='storage'
+                                        name='storage'
+                                        title={
+                                            <FormattedMessage
+                                                id='admin.sidebar.storage'
+                                                defaultMessage='Storage'
+                                            />
+                                        }
+                                    />
+                                ]}
```
Comment (jwilander) [inline webapp/components/admin_console/admin_sidebar.jsx]: Is there a reason for wrapping it in `{[]}`?

File: webapp/components/setting_picture.jsx [hunk 3]
```diff
@@ -114,8 +114,8 @@ export default class SettingPicture extends Component {
                                 id='setting_picture.help'
                                 defaultMessage='Upload a profile picture in BMP, JPG, JPEG or PNG format, at least {width}px in width and {height}px height.'
                                 values={{
-                                    width: global.mm_config.ProfileWidth,
-                                    height: global.mm_config.ProfileHeight
+                                    width: 128,
+                                    height: 128
```
Comment (jwilander) [inline webapp/components/setting_picture.jsx]: Maybe make constants for these as well

File: model/config.go [hunk 4]
```diff
@@ -231,12 +231,6 @@ type FileSettings struct {
 	Directory               string
 	EnablePublicLink        bool
 	PublicLinkSalt          *string
-	ThumbnailWidth          int
-	ThumbnailHeight         int
-	PreviewWidth            int
-	PreviewHeight           int
-	ProfileWidth            int
-	ProfileHeight           int
```
Comment (jwilander) [inline model/config.go]: You should also remove these from config.json

Comment (saturninoabril) [inline model/config.go]: noted.

File: app/file.go (hunk 1 above)
Comment (saturninoabril) [inline app/file.go]: noted.

File: webapp/components/setting_picture.jsx (hunk 3 above)
Comment (saturninoabril) [inline webapp/components/setting_picture.jsx]: noted.

File: webapp/components/admin_console/admin_sidebar.jsx (hunk 2 above)
Comment (saturninoabril) [inline webapp/components/admin_console/admin_sidebar.jsx]: Yes, children are required to be of type array.

Comment (hmhealey) [inline webapp/components/admin_console/admin_sidebar.jsx]: If you change the type of the children prop for the AdminSidebarSection component to `PropTypes.node`, you won't need the array. I'd prefer to do that over needing the array and the key here

File: webapp/components/setting_picture.jsx [hunk 5]
```diff
@@ -30,6 +30,8 @@ export default class SettingPicture extends Component {
         this.state = {
             image: null
         };
+
+        this.IMAGE_PROFILE_PIXEL_DIMENSION = 128;
```
Comment (hmhealey) [inline webapp/components/setting_picture.jsx]: This should actually go into `utils/constants.jsx`

Comment (saturninoabril) [inline webapp/components/setting_picture.jsx]: noted.

**Pull request merged**